CUAV_GPS AP_Periph build broken

## Bug report
Doesn't build.

**Version**
master

**Platform**

[  ] All
[  ] AntennaTracker
[  ] Copter
[  ] Plane
[  ] Rover
[  ] Submarine
[X] GPS 

**Airframe type**
AP_Periph

**Hardware type**
CUAV_GPS

**Logs**

```
./waf configure --board=CUAV_GPS
./waf build --target=bin/AP_Periph
```

[223/228] Compiling Tools/AP_Periph/can.cpp
../../Tools/AP_Periph/AP_Periph.cpp: In member function 'void AP_Periph_FW::init()':
../../Tools/AP_Periph/AP_Periph.cpp:114:16: error: 'class AP_HAL::RCOutput' has no member named 'set_neopixel_num_LEDs'; did you mean 'set_serial_led_num_LEDs'?
     hal.rcout->set_neopixel_num_LEDs(HAL_PERIPH_NEOPIXEL_CHAN, HAL_PERIPH_NEOPIXEL_COUNT);
                ^~~~~~~~~~~~~~~~~~~~~
compilation terminated due to -Wfatal-errors.

../../Tools/AP_Periph/can.cpp: In function 'void set_rgb_led(uint8_t, uint8_t, uint8_t)':
../../Tools/AP_Periph/can.cpp:510:16: error: 'class AP_HAL::RCOutput' has no member named 'set_neopixel_rgb_data'; did you mean 'set_serial_led_rgb_data'?
     hal.rcout->set_neopixel_rgb_data(HAL_PERIPH_NEOPIXEL_CHAN, -1, red, green, blue);
                ^~~~~~~~~~~~~~~~~~~~~
compilation terminated due to -Wfatal-errors.

--- a/Tools/AP_Periph/AP_Periph.cpp
+++ b/Tools/AP_Periph/AP_Periph.cpp
@@ -111,7 +111,7 @@ void AP_Periph_FW::init()
 
 #ifdef HAL_PERIPH_NEOPIXEL_COUNT
     hal.rcout->init();
-    hal.rcout->set_neopixel_num_LEDs(HAL_PERIPH_NEOPIXEL_CHAN, HAL_PERIPH_NEOPIXEL_COUNT);
+    hal.rcout->set_serial_led_num_LEDs(HAL_PERIPH_NEOPIXEL_CHAN, AP_HAL::RCOutput::MODE_NEOPIXEL);
 #endif
 
 #ifdef HAL_PERIPH_ENABLE_ADSB
@@ -157,8 +157,8 @@ static void update_rainbow()
     uint32_t now = AP_HAL::millis();
     if (now-start_ms > 1500) {
         rainbow_done = true;
-        hal.rcout->set_neopixel_rgb_data(HAL_PERIPH_NEOPIXEL_CHAN, -1, 0, 0, 0);
-        hal.rcout->neopixel_send();
+        hal.rcout->set_serial_led_rgb_data(HAL_PERIPH_NEOPIXEL_CHAN, -1, 0, 0, 0);
+        hal.rcout->serial_led_send(HAL_PERIPH_NEOPIXEL_CHAN);
         return;
     }
     static uint32_t last_update_ms;
@@ -186,13 +186,13 @@ static void update_rainbow()
     float brightness = 0.3;
     for (uint8_t n=0; n<8; n++) {
         uint8_t i = (step + n) % nsteps;
-        hal.rcout->set_neopixel_rgb_data(HAL_PERIPH_NEOPIXEL_CHAN, n,
+        hal.rcout->set_serial_led_rgb_data(HAL_PERIPH_NEOPIXEL_CHAN, n,
                                          rgb_rainbow[i].red*brightness,
                                          rgb_rainbow[i].green*brightness,
                                          rgb_rainbow[i].blue*brightness);
     }
     step++;
-    hal.rcout->neopixel_send();
+    hal.rcout->serial_led_send(HAL_PERIPH_NEOPIXEL_CHAN);
 }
 #endif
 
@@ -223,7 +223,7 @@ void AP_Periph_FW::update()
         show_stack_usage();
 #endif
 #ifdef HAL_PERIPH_NEOPIXEL_COUNT
-        hal.rcout->set_neopixel_num_LEDs(HAL_PERIPH_NEOPIXEL_CHAN, HAL_PERIPH_NEOPIXEL_COUNT);
+        hal.rcout->set_serial_led_num_LEDs(HAL_PERIPH_NEOPIXEL_CHAN, HAL_PERIPH_NEOPIXEL_COUNT, AP_HAL::RCOutput::MODE_NEOPIXEL);
 #endif
     }
     can_update();

--- a/Tools/AP_Periph/can.cpp
+++ b/Tools/AP_Periph/can.cpp
@@ -507,8 +507,8 @@ static void handle_RTCMStream(CanardInstance* ins, CanardRxTransfer* transfer)
 static void set_rgb_led(uint8_t red, uint8_t green, uint8_t blue)
 {
 #ifdef HAL_PERIPH_NEOPIXEL_COUNT
-    hal.rcout->set_neopixel_rgb_data(HAL_PERIPH_NEOPIXEL_CHAN, -1, red, green, blue);
-    hal.rcout->neopixel_send();
+    hal.rcout->set_serial_led_rgb_data(HAL_PERIPH_NEOPIXEL_CHAN, -1, red, green, blue);
+    hal.rcout->serial_led_send(HAL_PERIPH_NEOPIXEL_CHAN);
 #endif // HAL_PERIPH_NEOPIXEL_COUNT
 #ifdef HAL_PERIPH_ENABLE_NCP5623_LED
     {

